### PR TITLE
[EDX-70] Move Push REST API

### PR DESCRIPTION
**Pull request by tbedford**
> _NOTE TO REVIEWERS_ - please **do not review PRs in the `DRAFT` state**, as the PR may change substantially before it is ready to review. Thanks.

## Description

* [EDX-54](https://ably.atlassian.net/browse/EDX-54).
* [EDX-70](https://ably.atlassian.net/browse/EDX-70).

## Review

* [Push Notifications - Admin page](https://ably-docs-edx-70-move-p-zs4jcq.herokuapp.com/rest/push/)

### CODE REVIEW

**Review by m-hulbert — APPROVED** [review]

File: content/api/rest-sdk/push.textile [hunk 1]
```diff
@@ -0,0 +1,83 @@
+---
+title: Push
```
Comment (Srushtika) [inline content/api/rest-sdk/push.textile:2]: @tbedford @m-hulbert can we please sync up on the page titles? I thought we were going to match the titles we decided on in the Miro board?

Comment (tbedford) [reply content/api/rest-sdk/push.textile:2]: I'm looking at this (which is linked to from the DR linked from the epic): 
image
Although we did decide to rename the Encryption object to Crypto as that is what the object actually is.

Comment (m-hulbert) [reply content/api/rest-sdk/push.textile:2]: We have to revisit the pages that we'd completed from before that meeting too, so I'm expecting we'll create a "unify titles" ticket to tidy these up once we've completed REST and realtime.

Comment (m-hulbert) [reply content/api/rest-sdk/push.textile:2]: I think you're looking at the conceptual part in that screenshot @tbedford - the API reference is further down.

Comment (tbedford) [reply content/api/rest-sdk/push.textile:2]: Quite right. Although the names used there more or less correspond. @Srushtika can we sync up on how we should be naming again please, thanks.

File: content/api/versions/v1.1/rest-sdk/push-admin.textile [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+  - objc
+jump_to:
+  Push Admin API properties:
+    - deviceRegistrations#device-registrations
+    - channelSubscriptions#channel-subscriptions
+  Push Admin API methods:
+    - publish
+  Types:
+    - Related Types#related-types
+redirect_from:
```
Comment (m-hulbert) [inline content/api/versions/v1.1/rest-sdk/push-admin.textile:24]: The redirect for v1.1 gives a 404. I think it has to contain the full version path to redirect correctly for older versions: `/api/rest-sdk/versions/v1.1/push`. I'd suggest maybe we just remove it and only have a redirect on the latest. WDYT?

**Review by m-hulbert — CHANGES_REQUESTED** [review]

File: content/api/versions/v1.1/rest-sdk/push-admin.textile (hunk 2 above)
Comment (tbedford) [reply content/api/versions/v1.1/rest-sdk/push-admin.textile:24]: I think that makes sense!

**Review by m-hulbert — APPROVED** [review]

**Pull request merged**